src/server/state.rs

```diff
@@ -170,7 +170,7 @@ impl State {
         log::debug!("Task id={} updated {:?}", msg.id, msg.state);
         let job = self.get_job_mut_by_tako_task_id(msg.id).unwrap();
         match msg.state {
-            TaskState::Running => job.set_running_state(msg.id),
+            TaskState::Running(worker) => job.set_running_state(msg.id,worker),
```

Comment:
Let's keep it worker_id just for clarity :)

Reply:
plus running cargo_fmt should automatically add the missing whitespace after the comma